packages/client/src/internal/conversions/modernToLegacyAggregationClause.ts

```diff
@@ -45,7 +40,15 @@ export function modernToLegacyAggregationClause<
 
       return [
         {
-          type: metric as StringAggregateOption | NumericAggregateOption,
+          type: metric as
+            | "approximateDistinct"
```

Comment:
Is it not better if we had these aggregateoptions as a shared type like before? In case we need to modify/add an option?

Reply:
Sadly adding an option isn't going to do anything because you can always make this cast: https://www.typescriptlang.org/play/?#code/DYUwLgBAzgXNYCcCWA7A5gbgFBahAvBABQCsEApBAEwH6ECMAlBAPwQBEAFkuxHOwCMAnmBDtsWAMYB7FFEgAzadP4ppABzEFoEAIZ52azeyA

So really we are exposing types we have to keep back compat on that aren't serving us any good.

Reply:
ah ok makes sense